### ISSUE TITLE
Private repository when accepting assignment

### ISSUE THREAD

**Issue by jrwrigh** [cucs-hpsc/hpsc-community#2]

**Comment by jrwrigh** [comment]
It looks like using the "accept assignment" link creates a new public repo instead of a private repo like the previous times. Is this intentional?

**Comment by jedbrown** [comment]
Yes, it was intentional to facilitate discussions and peer evaluation. Each of the projects are distinct.

**Comment by jrwrigh** [comment]
Cool!
